.github/workflows/pre-release.yml

```diff
@@ -285,6 +289,30 @@ jobs:
         run: while ! curl -s 'http://localhost:3000/api/health' | grep '{"status":"ok"}'; do sleep 1; done
         timeout-minutes: 3
 
+  trigger-auto-publish:
+    if: ${{ inputs.auto }}
```

Comment:
iirc we need to compare with strings like `inputs.auto != 'false'`, but maybe `default` makes it differently

https://stackoverflow.com/a/76294014

Reply:
the types on these github action params are 🥴 

I ran another test on this workflow, and it's working as intended.

[run without auto flag](https://github.com/iethree/metabase/actions/runs/10199803482):  skips auto-publish
[run with auto flag](https://github.com/iethree/metabase/actions/runs/10152952272): triggers auto-publish